helm/kubernetes-prometheus-chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+description: A Helm Chart for Prometheus Operator.
+icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
+name: kubernetes-prometheus-chart
```

Comment:
Chart name is prefixed with repo name so https://github.com/giantswarm/architect-orb/blob/master/src/commands/package-and-push.yaml#L12 check should pass.

@kopiczko is "-chart" suffix supported in architect / architect-orb?

Reply:
@sslavic The `-chart` suffix isn't considered anymore. The validation was loosened by Pawel here. https://github.com/giantswarm/architect/pull/316

See my main comment for what I think we should use for the repo and chart names.

Reply:
but then it is ok. If we want to make `helm lint` work too the folder and chart name should match

Reply:
I think we can make them match. I've had an idea on how we should do the naming. I'm going to write it up in a discuss ticket later today.

So for now, let's just get this working and not worry about it too much.